app/src/main/java/org/simple/clinic/recentpatient/RecentPatientsScreen.kt

```diff
@@ -0,0 +1,98 @@
+package org.simple.clinic.recentpatient
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.LinearLayout
+import androidx.appcompat.widget.Toolbar
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.jakewharton.rxbinding2.view.RxView
+import com.xwray.groupie.GroupAdapter
+import com.xwray.groupie.ViewHolder
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.rxkotlin.ofType
+import io.reactivex.subjects.PublishSubject
+import kotterknife.bindView
+import org.simple.clinic.R
+import org.simple.clinic.activity.TheActivity
+import org.simple.clinic.router.screen.ScreenRouter
+import org.simple.clinic.summary.OpenIntention
+import org.simple.clinic.summary.PatientSummaryScreenKey
+import org.simple.clinic.util.UtcClock
+import org.simple.clinic.widgets.ScreenCreated
+import org.simple.clinic.widgets.ScreenDestroyed
+import org.simple.clinic.widgets.UiEvent
+import org.threeten.bp.Instant
+import java.util.UUID
+import javax.inject.Inject
+
+class RecentPatientsScreen(context: Context, attrs: AttributeSet) : LinearLayout(context, attrs) {
```

Comment:
If we are going to make this screen accessible once the PR is merged, we also need to provide tests for the controller. Can you either:
- Add tests for the controller, or
- Don't show the `See All` button yet and add it along with the PR that adds the tests?

Reply:
I'll be adding code to show "See all" button in a following PR.